.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Setup Java JDK
       uses: actions/setup-java@v1.4.3
       with:
-        java-version: 11
+        java-version: 17
```

Comment:
@jackra1n  any reasons why you didn't change to our reusable pipelines?

Reply:
I didnt think about it 🤔. I dont think we talked about those and I didn't want to break anything 😬 
If I finish my sprint tasks quickly and there wont be anything else important I can help with, I will work on switching those to our reuseable pipelines

Reply:
I'll do it.